app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -315,7 +315,7 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
         status={Status.SUCCESS}
         className={
           successMessage === fileLoadingText || successMessage === fileLoadedText
-            ? 'usa-sr-only'
+            ? 'usa-sr-only display-none'
```

Comment:
Does `display-none` affect whether this message is announced to screen readers?

Reply:
Um. Yeah. Addressing.

Reply:
Looks like the right solution is to remove the sr-only class if the message is fileLoadedText (so the space is not empty, but has a nice checkbox and "Image uploaded" text. Updated.